### PR TITLE
Docs 7727 address mq 1887 custom headers hn

### PR DESCRIPTION
**Pull request by hannanelson**
Here's a PDF of the topics that changed (Queues, Exchanges, Tutorial).
[MQ-User-Properties-Updates.pdf](https://github.com/mulesoft/docs-mq/files/4495725/MQ-User-Properties-Updates.pdf)

### CODE REVIEW

**Review by danieldatri — DISMISSED** [review]

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -127,6 +127,15 @@ To preserve message order for an app, you might want to create a message group f
 image::mq-fifo-msg-groups.png[Message Group ID field for FIFO queues in the Message Sender page]
 +
 See <<FIFO Queues and Message Groups>>.
+** *Add User Properties* configures user properties to send with the message.
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc:130]: Please verify that this should be a Level-2 bullet?

File: modules/ROOT/pages/mq-queues.adoc [hunk 2]
```diff
@@ -127,6 +127,15 @@ To preserve message order for an app, you might want to create a message group f
...
++
+Toggle *Add User Properties* and specify the property name-value pairs to send:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc:132]: Should this be a step?

File: modules/ROOT/pages/mq-queues.adoc [hunk 3]
```diff
@@ -174,9 +183,8 @@ Deleting a message is an acknowledgment operation (ACK). For information about h
 Returning messages to the queue is a not-acknowledgment operation (NACK). The messages aren't altered. However, the time-to-live (TTL) value you set when you created the queue determines how long the message is available before Anypoint MQ deletes it.
 +
 If you switch from the *Message Browser* page, messages automatically return to the queue.
-
-
-If user properties are configured in the Publish operation in Anypoint MQ Connector, they appear in the details pane for the message:
++
+If user properties are configured in the *Message Sender* page or Publish operation in Anypoint MQ Connector, they appear in the details pane for the message:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]:
```suggestion
If user properties are configured in the *Message Sender* page or as part of the Publish operation in Anypoint MQ Connector, they appear in the details pane for the message:
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 4]
```diff
@@ -89,7 +89,12 @@ image::mq-tutorial-message-sender-page.png[Message Sender option]
 . Enter `Hello Mules!` in the *Payload* field.
 +
 Leave the *Type* field set to *Text*.
+. If you want, toggle *Add User Properties* and specify the property name-value pairs to send.
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
. If you want, toggle *Add User Properties* and specify the property name-value pairs to send:
```

**Review by kbostrom — DISMISSED** [review]
Copyedited

File: modules/ROOT/pages/mq-queues.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc:130]: Yes

File: modules/ROOT/pages/mq-queues.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc:132]: No - it's part of the bullet list of options on the page.

**Review by prafulrana — DISMISSED** [review]

**Review by danieldatri — APPROVED** [review]

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**